dags/resources/stages/parse/table_definitions/hop_protocol_usdc_bridge/L1_ERC20_Bridge_event_Withdrew.json

```diff
@@ -0,0 +1,65 @@
+{
+    "parser": {
+        "abi": {
+            "anonymous": false,
+            "inputs": [
+                {
+                    "indexed": true,
+                    "internalType": "bytes32",
+                    "name": "transferId",
+                    "type": "bytes32"
+                },
+                {
+                    "indexed": true,
+                    "internalType": "address",
+                    "name": "recipient",
+                    "type": "address"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "uint256",
+                    "name": "amount",
+                    "type": "uint256"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "bytes32",
+                    "name": "transferNonce",
+                    "type": "bytes32"
+                }
+            ],
+            "name": "Withdrew",
+            "type": "event"
+        },
+        "contract_address": "0x3666f603cc164936c1b87e207f36beba4ac5f18a",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "hop_protocol_usdc_bridge",
```

Comment:
Let's just rename it to `hop_protocol` for brevity. Folder name should be `hop_protocol` too

Reply:
mbad ser - there's actually a bunch of token bridging events - not sure what's the best way to do it 🙏

Reply:
The rest here - apologies, should have done it all tgt

https://github.com/blockchain-etl/ethereum-etl-airflow/pulls

Reply:
Hey @darrenlimweiyang if the contracts all have the same event signatures there is a way to add multiple contract addresses to the definition, e.g.

```
"contract_address": "SELECT * FROM UNNEST(['0x7e6d3b1161df9c9c7527f68d651b297d2fdb820b', '0x9134c9975244b46692ad9a7da36dba8734ec6da3'])"
```

See example here https://github.com/blockchain-etl/ethereum-etl-airflow/blob/master/dags/resources/stages/parse/table_definitions/aave/AToken_event_InterestRedirectionAllowanceChanged.json

Reply:
Noted @medvedev1088  ty ser! I'll just close this PRs and open a new one?

Reply:
> Noted @medvedev1088 ty ser! I'll just close this PRs and open a new one?

Sounds good, thanks!